src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -406,6 +391,7 @@ public string UserAgent
         /// <summary>
         /// Whether JavaScript is enabled.
         /// </summary>
+        /// <since_tizen> 8 </since_tizen>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public bool EnableJavaScript
```

Comment:
In my opinion, JavaScriptEnabled is better.
(https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members)

Reply:
Done.